lib/SellecktPopup.js

```diff
@@ -319,7 +319,8 @@ _.extend(SellecktPopup.prototype, {
             openerOffset = $opener.offset(),
             viewportBottom = windowScrollTop + windowHeight,
             enoughRoomBelow = viewportBottom > (openerOffset.top + openerHeight + popupHeight),
-            showBelow = enoughRoomBelow && !(options.keepCurrentOrientation && this.$popup.hasClass('flipped')),
+            enoughRoomAbove = openerOffset.top >= popupHeight,
+            showBelow = (enoughRoomBelow || !enoughRoomAbove) && !(options.keepCurrentOrientation && this.$popup.hasClass('flipped')),
```

Comment:
Can you split this out into a function? It's pretty ugly with the extra condition.